stock_operating_unit/security/stock_security.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 Eficent Business and IT Consulting Services S.L.
+     Copyright 2016-2017 Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo noupdate="1">
+    <record id="ir_rule_stock_warehouse_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="stock.model_stock_warehouse"/>
+        <field name="domain_force">['|',('operating_unit_id','=',False),('operating_unit_id','in',[g.id for g in user.operating_unit_ids])]</field>
```

Comment:
Yes, as I said, the code and the xml file of permissions seem good. The records in the database are the same, but I see the store that creates the installation when the company is created. The following that are created works ok.

It only occurs to me that the permissions of another module are affecting ...

Note: I did not answer before because github did not notify me of this review ...